### PR TITLE
Adding documentation about Extended Data fields in the response

### PR DESCRIPTION
**Pull request by lmilisits**
Email requires Extended Data Access in the Environment response,
otherwise email field is null.

### CODE REVIEW

**Comment by msgerbush** [comment]
We might want to try to change the code blocks to be `js` instead of `json`, so that the comments don't show up as syntax errors with the red highlighting.  Let me know if that works.

**Comment by lmilisits** [comment]
Updated (hopefully correctly)!

File: source/documentation/cloud-apps/reference/helpdesk.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Example ticket (note all arrays have been reduced to a single example item):
       "show_url": "/people/2"
     }
   ],
-  "comments": [
+  "comments": [  // Extended Data Access required, else not returned
```
Comment (msgerbush) [inline source/documentation/cloud-apps/reference/helpdesk.md]: looks like you missed one, this is still marked as a syntax error.

File: source/documentation/cloud-apps/reference/people.md [hunk 2]
```diff
@@ -56,11 +56,11 @@ Example person (note all arrays have been reduced to a single example item):
 ```json
 {
   "id": 13,
-  "first_name": "Alexia",
-  "last_name": "Rocoto",
+  "first_name": "Alexia", // Extended Data Access required, else not returned
```
Comment (msgerbush) [inline source/documentation/cloud-apps/reference/people.md]: looks like these three are still not parsing right.

**Comment by msgerbush** [comment]
:+1: Thanks!

**Pull request merged**